packages/compass-import-export/src/components/import-modal.tsx

```diff
@@ -61,11 +65,34 @@ const closeButtonStyles = css({
   marginRight: spacing[2],
 });
 
+const progressStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  backgroundColor: palette.gray.light3, // TODO: dark mode
```

Comment:
Is it passable in dark mode without this? If not, maybe we can come up with some color here before merging?

Reply:
aaargh I forgot about dark mode

Reply:
![Screenshot 2023-03-27 at 12 17 41](https://user-images.githubusercontent.com/69737/227926815-100be6b6-5b0d-443e-a23f-0f9d35dc6474.png)